test/Microsoft.ML.Benchmarks/PredictionEngineBench.cs

```diff
@@ -58,7 +58,7 @@ public void SetupIrisPipeline()
             var pipeline = new ColumnConcatenatingEstimator(env, "Features", new[] { "SepalLength", "SepalWidth", "PetalLength", "PetalWidth" })
                 .Append(env.Transforms.Conversion.MapValueToKey("Label"))
                 .Append(env.MulticlassClassification.Trainers.SdcaMaximumEntropy(
-                    new SdcaMaximumEntropyMulticlassTrainer.Options { NumberOfThreads = 1, ConvergenceTolerance = 1e-2f, }));
+                    new SdcaMaximumEntropyMulticlassTrainer.Options { NumberOfThreads = 1, ConvergenceTolerance = 1e-2f, Shuffle = false }));
```

Comment:
I'm unsure if disabling shuffling with cause the unit tests to be more stable. The shuffling and TrainTest/CV splits (hopefully) listen to seed, and would then be deterministic. 

Can verify by inspecting the one of the TrainTest splits, or repeatedly dumping the iDataView to a file (expecting with a fixed seed they stable perfectly the same, and changing the seed gets you a different set and ordering of rows).

Specifically, I would first test if shuffling is already stable given a seed, before making this change. Shuffling is beneficial to have turned on.

Copied from: https://github.com/dotnet/machinelearning/pull/4736#issuecomment-580515353

Reply:
I agree with Justin. We just need to check the final value to be within certain range. Shuffling is important for SDCA.

Reply:
So shuffle seed is indeed coming from `MLContext`:
https://github.com/dotnet/machinelearning/blob/24c827416d23a6a74f16f71ec9123ca15e0e12fd/src/Microsoft.ML.StandardTrainers/Standard/SdcaBinary.cs#L555-L557

`Host` is being registered from the environment here:
https://github.com/dotnet/machinelearning/blob/24c827416d23a6a74f16f71ec9123ca15e0e12fd/src/Microsoft.ML.StandardTrainers/Standard/SdcaBinary.cs#L300-L302
https://github.com/dotnet/machinelearning/blob/24c827416d23a6a74f16f71ec9123ca15e0e12fd/src/Microsoft.ML.Core/Environment/HostEnvironmentBase.cs#L385-L392

and `_rand` is getting its seed from `MLContext`:
https://github.com/dotnet/machinelearning/blob/24c827416d23a6a74f16f71ec9123ca15e0e12fd/src/Microsoft.ML.Core/Environment/HostEnvironmentBase.cs#L350-L355
https://github.com/dotnet/machinelearning/blob/24c827416d23a6a74f16f71ec9123ca15e0e12fd/src/Microsoft.ML.Data/Utilities/LocalEnvironment.cs#L49-L50
https://github.com/dotnet/machinelearning/blob/24c827416d23a6a74f16f71ec9123ca15e0e12fd/src/Microsoft.ML.Data/MLContext.cs#L110-L115

Reply:
`TrainTestSplit`, `CrossValidationSplit`, and `CrossValidate` have a seed, but when not specified, the seed doesn't come from `MLContext`. Instead, `GenerateNumberTransform` and `HashingEstimator` use their own default seeds. See #4752